packages/client/hmi-client/src/main.ts

```diff
@@ -20,22 +20,20 @@ import App from '@/App.vue';
 import { useProjects } from '@/composables/project';
 import { useNotificationManager } from '@/composables/notificationManager';
 import '@/assets/css/style.scss';
-import Keycloak from 'keycloak-js';
+import { createOidc } from 'oidc-spa';
 import { init as clientEventServiceInit } from '@/services/ClientEventService';
 
-// Extend the window object to include the Keycloak object
-declare global {
-	interface Window {
-		keycloak_init: Promise<boolean>;
-		keycloak: Keycloak;
-	}
-}
+const oidcSettings = await fetch('/api/configuration/keycloak').then((r) => r.json());
+const oidc = await createOidc({
+	issuerUri: `${oidcSettings['auth-server-url']}/realms/${oidcSettings.realm}`,
+	clientId: oidcSettings.resource,
+	publicUrl: '/'
+});
```

Comment:
Why this is not done within `useAuthStore.init` method?

Reply:
A great question, before a lot was littered within `main.ts` - I will look into tiding this up